include/RCubeViewer/RCubeViewer.h

```diff
@@ -73,6 +73,8 @@ class RCubeViewer : public rcube::Window
                                int numSegments);
     EntityHandle addSurface(const std::string name, const MeshData &data);
 
+    EntityHandle addOBJ(const std::string name, const MeshData &data);
```

Comment:
Let's not have this as a separate method. We already have `addSurface()` method above, the user just needs to call `addSurface("some_name", loadOBJ(filename));`